xsd/netex_framework/netex_reusableComponents/netex_serviceRestrictions_version.xsd

```diff
@@ -132,15 +134,52 @@ Rail transport, Roads and Road transport
 		</xsd:sequence>
 	</xsd:group>
 	<!-- ======================================================================= -->
-	<xsd:complexType name="BookingArrangementsStructure" abstract="false">
+	<xsd:element name="BookingArrangement" abstract="true" substitutionGroup="DataManagedObject">
```

Comment:
```suggestion
	<xsd:element name="BookingArrangement" substitutionGroup="DataManagedObject">
```

Reply:
@Aurige this must have been a technical mistake?

Reply:
right, good catch

Reply:
@Aurige did you apply it? Because I cannot see commits after mine, and I cannot apply it anymore.